README.md

```diff
@@ -633,6 +633,10 @@ We are still in progress of refactoring some methods, more details to be updated
 
 Currently `update` does the same thing as `replace`, for details, refer to https://github.com/strongloop/loopback-connector-cloudant#no-partial-update
 
+## Find By Id
+
+See the advanced queries documentation [Find By Id](./doc/find-by-id.md)
```

Comment:
Is it worth creating a new page for those ~20 lines? Is this link going to work when viewed at https://www.npmjs.com/package/loopback-connector-cloudant?

Reply:
sure I can move it back to the readme :p